fud2/fud-core/src/script/plugin.rs

```diff
@@ -12,35 +13,35 @@ use super::{
     error::RhaiSystemError,
     exec_scripts::{to_rhai_err, to_str_slice, RhaiResult, RhaiSetupCtx},
     report::RhaiReport,
+    resolver::Resolver,
 };
 
+#[derive(Clone)]
 struct ScriptContext {
-    builder: DriverBuilder,
-    path: PathBuf,
-    ast: rhai::AST,
+    builder: Rc<RefCell<DriverBuilder>>,
+    path: Rc<PathBuf>,
+    ast: Rc<rhai::AST>,
 }
```

Comment:
This is a much better arrangement of the `Rc`s than what I had; thanks for working that out.